README.md

```diff
@@ -1,4 +1,5 @@
-![Image of PixelFlasher Icon](/images/icon-256.png)
+<img src="https://github.com/badabing2005/PixelFlasher/blob/main/images/icon-256.png" width="128" height="128" />
```

Comment:
The recommended way is to reference as I have done, you're doing this because you want to show it smaller?
Perhaps it is better to include a 128x128 version in the images directory and reference that.

Reply:
I guess it does not matter if I add a 128px raster or the browser scales it down.